Frends.Kafka.Consume/Frends.Kafka.Consume.Tests/TestBasic.cs

```diff
@@ -0,0 +1,1167 @@
+using Confluent.Kafka;
+using Frends.Kafka.Consume.Definitions;
+using NUnit.Framework;
+using NUnit.Framework.Legacy;
+
+namespace Frends.Kafka.Consume.Tests;
+
+[TestFixture]
+public class TestBasic
+{
+    /*
+        Update 05/2024:
+            Using cloud Confluent Kafka. Set your own configs or see 'Basic testing with docker'. 
+    
+        Basic testing with docker:
+        Docker compose:
+        Run command 'docker-compose up -d' in .\Frends.Kafka.Consume.Tests
+        
+        Read message(s) from topic:
+        docker exec --interactive --tty "container's name" kafka-console-consumer --bootstrap-server localhost:9092 --topic ConsumeTopic --from-beginning
+    */
+
+    private readonly string _message = $"Hello {DateTime.Now}";
+    private readonly string _topic = "ConsumeBasicTopic";
+    private readonly string? _bootstrapServers = Environment.GetEnvironmentVariable("ConfluentKafka_BootstrapServers");
+    private readonly string? _apiKey = Environment.GetEnvironmentVariable("ConfluentKafka_APIKey");
+    private readonly string? _apiKeySecret = Environment.GetEnvironmentVariable("ConfluentKafka_APIKeySecret");
+    private readonly string? _schemaRegistryURL = Environment.GetEnvironmentVariable("ConfluentKafka_SchemaRegistryURL");
+    private readonly string? _schemaRegistryAPIKey = Environment.GetEnvironmentVariable("ConfluentKafka_SchemaRegistryAPIKey");
+    private readonly string? _schemaRegistryAPIKeySecret = Environment.GetEnvironmentVariable("ConfluentKafka_SchemaRegistryAPIKeySecret");
+
+    private Input _input = new();
+    private Options _options = new();
+    private Sasl _sasl = new();
+    private Socket _socket = new();
+    private Ssl _ssl = new();
+    private SchemaRegistry _schemaRegistry = new();
+
+    private readonly string _records = @"{
+  ""intField"": 1,
+  ""longField"": 1234567890,
+  ""floatField"": 1.23,
+  ""doubleField"": 1.23456789,
+  ""booleanField"": true,
+  ""stringField"": ""Hello, World!"",
+  ""nullField"": null,
+  ""bytesField"": ""dGVzdDE="",
+  ""enumField"": ""RED"",
+  ""arrayField"": [""item1"", ""item2"", ""item3""],
+  ""mapField"": {
+    ""key1"": 1,
+    ""key2"": 2,
+    ""key3"": 3
+  },
+  ""fixedField"": ""YWJjZA=="",
+  ""unionField"": ""Hello, Union!"",
+  ""recordField"": {
+    ""nestedField"": ""Hello, Nested!""
+  }
+}
+";
+    private readonly string _schema = @"{
```

Comment:
Can you check why github actions is giving those warnings about _schema not used?

Reply:
Forgot to remove _records  and _schema from TestBasic.cs. Those are not needed when running "basic" tests.